.github/workflows/ci-build.yaml

```diff
@@ -14,6 +14,11 @@ jobs:
   lint_and_typecheck:
     runs-on: ubuntu-latest
     steps:
+      - name: Cancel previous
+        uses: styfle/cancel-workflow-action@0.7.0
+        with:
+          access_token: ${{ github.token }}
+        if: ${{github.ref != 'refs/head/main'}}
```

Comment:
Does `refs/head/main` refer to a branch name? If so I think this should probably be `refs/head/master` for this repository. Am I understanding that correctly?

Reply:
oop - good catch. Fixed!

Reply:
It looks like there are two other locations as well. Thanks!

Reply:
Sorry - I'm working across 50 repos and E_TOO_MANY_TABS. Updated.